Remotion/Web/Development.WebTesting/HostingStrategies/AspNetCoreHostingProcessWrapper.cs

```diff
@@ -58,7 +65,9 @@ private Process CreateServerProcess ()
     if (OperatingSystem.IsWindows())
       startInfo.LoadUserProfile = true;
 
-    startInfo.FileName = _exePath;
+    startInfo.FileName = _processPath;
+    if (_processArguments != null)
+      startInfo.Arguments = _processArguments;
```

Comment:
The default for StartInfo.Arguments is an empty string. shouldn't we just use _processArguments??"" instead of the IF? Or make the process arguments not-nullable for consistency, same as the processpath. Note that further up the callchain, processpath is nullable so we don't just change this in the config-part of the code, but only later on perhaps?